packages/webpack/test/e2e-scuttle.spec.js

```diff
@@ -0,0 +1,88 @@
+const test = require('ava')
```

Comment:
if you want strict types for your test context, you could do something like

```js
// eslint-disable-next-line ava/use-test
const { default: ava } = require('ava')

/**
 * Test context object for our tests
 *
 * @typedef ScuttlingTestContext
 * @property {type} x
 * @property {type} y
 * @property {type} z
 */

const test = /** @type {import('ava').TestFn<ScuttlingTestContext>} */(ava)
```

Reply:
Pretty cool. Do I want this though?

Reply:
It could suggest correct assertions fromt the `t` reference inside the test case (As opposed to copilot, which often makes up assertions that don't exist)